search/tests/test_indexing.py

```diff
@@ -101,10 +110,15 @@ def test_sorting(self):
         )
         # expected position (res2, res3, res1)
         top_res = results[0]
+        sec_res = results[1]
         self.assertEqual(
             top_res.nr_views,
             res2.xa_nr_views
         )
+        self.assertEqual(
```

Comment:
I think these tests don't test that the results appear in the proper order, because it's comparing the column that is identical between the two. The foreign key would be better (at least asserting that it's lower in `top_res` than in `sec_res`.